scio-examples/src/main/scala/com/spotify/scio/examples/extra/WriteDynamicExample.scala

```diff
@@ -0,0 +1,67 @@
+/*
+ * Copyright 2018 Spotify AB.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package com.spotify.scio.examples.extra
+
+import com.spotify.scio.ContextAndArgs
+import com.spotify.scio.examples.common.ExampleData
+import org.apache.beam.sdk.coders.StringUtf8Coder
+import org.apache.beam.sdk.io.{FileIO, TextIO}
+import org.apache.beam.sdk.transforms.Contextful
+
+// `sbt runMain "com.spotify.scio.examples.extra.WriteDynamicExample
```

Comment:
Add "Example" and "Usage" likes like here?
https://github.com/spotify/scio/blob/master/scio-examples/src/main/scala/com/spotify/scio/examples/WordCount.scala#L18

Also these are in the generated socco web pages. So move all of them to above `package`? So that the generated page formats nicely.

http://spotify.github.io/scio/examples/WordCount.scala.html